POLMED/src/App.tsx

```diff
@@ -1,13 +1,17 @@
-import { Routes, Route } from "react-router-dom";
+import { Routes, Route, useLocation } from "react-router-dom";
 import Home from "@/pages/home";
 import OurDoctors from "@/pages/ourDoctors";
 import Pricing from "@/pages/pricing";
 import Contact from "@/pages/contact";
-import MyAccount from "./pages/myAccount";
+import MyAccount from "@/pages/myAccount";
 import Nav from "@/components/nav";
+import HomePageFooter from "./components/footers/home-page";
+import OtherPageFooter from "./components/footers/other";
 import "@/styles/main.scss";
 
 const App = () => {
+  const location = useLocation();
+  const atHomePage = location.pathname === "/";

```

Comment:
Zastanawiam się czy nie utworzyć globalnego kontekstu który przechowuje aktualną ścieżkę w naszej aplikacji, aby nie musieć korzystać z useLocation za każdym razem gdy potrzebujemy uzyskać dostęp do ścieżki. Wówczas korzystalibyśmy tylko z naszego customowego hooka, który by przechowywał dokładną ścieżkę bazując właśnie na useLocation

Reply:
Oczywiście to w sytuacji, gdy zaakceptujemy to rozwiązanie z architekturą footera